Yosemite/Yosemite/Stores/ReceiptStore.swift

```diff
@@ -93,9 +93,24 @@ private extension ReceiptStore {
     }
 
     func generateLineItems(order: Order) -> [ReceiptLineItem] {
-        order.items.map { item in
-            ReceiptLineItem(
-                title: item.name,
+        order.items.map {item in
+            var attributesText = ""
+            if !item.attributes.isEmpty {
```

Comment:
I would suggest passing the attributes as an array to ReceiptLineItem and then let the entity rendering the receipt take care of actually generating the lines of text that correspond to the attributes

Reply:
Good point. Done here 38cbb17ee811d91928439e1aedd76b73 and a test 974fabe25138583e4697129ae237649536011dea